README.md

```diff
@@ -30,48 +30,91 @@ Options:
+- [Offline](#offline) - No connection to upstream node. Full control of data and tokens;
 
-You have a need to have full control of both the data your clients will get and which keys can be used to access the server. This mode needs a downloaded JSON dump of a result from a query against an Unleash server on the [/api/client/features](https://docs.getunleash.io/reference/api/unleash/get-client-feature) endpoint as well as a comma-separated list of keys that should be allowed to access the server.
+#### Edge
 
-If your keys follow the Unleash API key format `[project]:[environment].<somesecret>`, Edge will filter the features dump to match the project contained in the key.
+```mermaid
+graph LR
+  A(Client) -->|Fetch toggles| B((Edge))
+  B-->|Fetch features| C((Unleash))
+```
+
+Edge mode is the "standard" mode for Unleash Edge and the one you should default to in most cases. It connects to an upstream node, such as your Unleash instance, and uses that as the source of truth for feature toggles.
 
-If you'd rather use a simple key like `secret-123`, any query against `/api/client/features` will receive the dump passed in on the command line.
+Another than connecting Edge directly to your Unleash instance, it's also possible to connect to another Edge instance (_daisy chaining_). You can have as many Edge nodes as you'd like between the Edge node your clients are accessing and the Unleash server, and it's also possible for multiple nodes to connect to a single upstream one. Depending on your architecture and requirements this can be a powerful feature, offering you flexibility and scalability when planning your implementation.
 
-Any query against `/api/frontend` or `/api/proxy` with a valid key will receive only enabled toggles.
+```mermaid
+graph LR
+  A(Client 1) -->|Fetch toggles| C((Edge 1))
+  B(Client 2) -->|Fetch toggles| D((Edge 2))
+  C-->|Fetch features| E((Edge 3))
+  D-->|Fetch features| E
+  E-->|Fetch features| F((Unleash))
+```
+
+This means that, in order to start up, Edge mode needs to know where the upstream node is. This is done by passing the `--unleash-url` command line argument or setting the `UNLEASH_URL` environment variable.
+
+By default, Edge mode uses in-memory to store the features it fetches from the upstream node. However, you may want to use a more persistent storage solution. Edge supports Redis as well, and you can configure it by passing in the `--redis-url` command line argument or setting the `REDIS_URL` environment variable.
+
+Edge mode also supports dynamic tokens, meaning that when using a [client token](https://docs.getunleash.io/reference/api-tokens-and-client-keys#client-tokens) with the `/api/client/features` endpoint, even without previously knowing about it, Edge will validate upstream and fetch its respective features. After that, it gets added to the list of known tokens that gets periodically synced, making sure it is a valid token and its features are up-to-date.
```

Comment:
I'd add a couple of notes:
1. Edge does not need a token to start
2. It might be useful to provide an initial token in order to warm it up (and how to do it)

Reply:
Good points, addressed in https://github.com/Unleash/unleash-edge/pull/60/commits/970533f963717d32a29c3b3a78a8dd7b11430ad2